www/js/intro.js

```diff
@@ -109,7 +109,11 @@ angular.module('emission.intro', ['emission.splash.startprefs',
       $scope.userEmail = userEmail;
       CommHelper.registerUser(function(successResult) {
           const uuid = successResult.uuid;
+<<<<<<< HEAD
```

Comment:
This has a merge conflict.

Reply:
I thought I have resolve that before creating PR. However, I just did now.